internal/kldcontracts/smartcontractgw.go

```diff
@@ -650,6 +650,7 @@ func (g *smartContractGW) createStream(res http.ResponseWriter, req *http.Reques
 		g.gatewayErrReply(res, req, klderrors.Errorf(klderrors.RESTGatewayEventStreamInvalid, err), 400)
 		return
 	}
+	log.Warn(spec.WebSocket)
```

Comment:
Is this deliberately left, or lint from previous debugging?

Reply:
Good catch. That was left from debugging, sorry. I missed that when I reviewed the diff. I will remove.